examples/bastion-tunnel-monorepo/.ize/env/testnut/ize.toml

```diff
@@ -0,0 +1,10 @@
+aws_profile = "nutcorp-dev"
+aws_region = "us-east-1"
+env = "testnut"
+namespace = "testnut"
+terraform_version = "1.1.7"
+
+[infra.terraform]
+aws_profile = "nutcorp-dev"
+aws_region = "us-east-1"
+terraform_version = "1.1.7"
```

Comment:
We can remove this as we don't really need it at the moment (outer scope config would prevail)
```suggestion
``